src/Http/GraphRequest.php

```diff
@@ -129,7 +129,7 @@ public function __construct($requestType, $endpoint, $accessToken, $baseUrl, $ap
 
         $this->baseUrl = $baseUrl;
         $this->apiVersion = $apiVersion;
-        $this->timeout = 0;
+        $this->timeout = 100;
```

Comment:
Can a comment be added stating why value is set to 100?

Reply:
I copied the default of C# HttpClient:
https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.timeout?view=net-5.0#remarks

Reply:
I think that subscribing to notifications can depend on another client to respond to Graph so I suspect that could take sometime if a user was part of that critical path.